[plugin/feedback-entity] Improvements to Feedback Plugin UI (#22478)
backstage/backstage: .changeset/cyan-toes-repeat.md

---
'@backstage/plugin-entity-feedback-backend': patch
---

> Add in description for 400 response when rating and not authenticated

Reviewer: ```suggestion
Add in description for 400 response when encountering an invalid rating request
```
